### ISSUE TITLE
"Error reading file" during project load (minimum audio bitrate?)

### ISSUE THREAD

**Issue by ThereforeGames** [bscotch/stitch#121]
Hi,

Love the idea of using VSCode for my GameMaker projects, but unfortunately I'm having some trouble getting it to work. I'm encountering the following error related to one of my game's audio files:

```
ERROR 01:32:06.141 Error loading project demoncrawl.yyp Error: Error reading file: t:/code/gamemaker/demoncrawl/sounds/snd_halloween_secret_a/snd_halloween_secret_a.yy
[
  {
    "code": "too_small",
    "minimum": 128,
    "type": "number",
    "inclusive": true,
    "exact": false,
    "message": "Number must be greater than or equal to 128",
    "path": [
      "bitRate"
    ]
  }
]
```

The .yy file is attached. Any idea what's causing this?

Thanks!

[snd_halloween_secret_a.zip](https://github.com/bscotch/stitch/files/12297391/snd_halloween_secret_a.zip)

**EDIT:** Oh duh, it's the bitrate of this particular audio file. Why is < 128 throwing an error? The GM IDE generated this yy file and has no trouble compiling at 96 bitrate.

**Comment by adam-coster** [comment]
Whoops! That's an easy fix, though it may be a bit before I get the chance to make it.

Long story short:

- I have a `yy` file parser that does some validation, which is used to (1) test that my assumptions about the structure/contents of any given `yy` file type are correct, and (2) ensure that when Stitch edits `yy` files it doesn't break anything.
- The validator enforces that range, but I set it a *long* time ago. So either the minimum has changed since then, or something wasn't documented correctly, or I just made a mistake.
- All I need to do is change that one little spot in the validator to fix it.
